index.html

```diff
@@ -1,12 +1,336 @@
+            <div class="col-12">
+              <div class="text-center centered-title mb-5 pb-lg-4">
+                <h2 class="text-black">Some Dishes</h2>
+                <div class="line"></div>
+              </div>
+            </div>
+          </div>
+          <div id="get-dishes" class="row"></div>
+          <div class="row mt-3 d-md-none d-lg-none d-xl-none">
+            <div class="col-12">
+              <div class="call-to-action pt-lg-5 pt-xl-5">
+                <a
+                  id="show-more-dishes"
+                  class="show-more-dishes btn btn-outline-dark w-100 d-flex align-items-baseline justify-content-center"
+                  href="#"
+                >
+                  <div class="me-2">More</div>
+                  <i class="fa fa-angle-down orange-text"></i>
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- Partners -->
+
+    <section id="partners">
+      <div class="container-fluid bg-dark py-5 px-lg-5">
+        <h3 class="text-center text-white">Our Partners</h3>
+        <div class="row mt-5 px-5 mx-5">
+          <div
+            class="col-lg-2 pe-4 col-md-4 col-4 px-0 justify-content-center d-flex align-item-center"
+          >
+            <img class="foodie" src="images/partner2.svg" alt="" />
+          </div>
+
+          <div
+            class="col-lg-2 pe-4 col-md-4 col-4 px-0 justify-content-center d-flex align-item-center"
+          >
+            <img class="foodie" src="images/partner3.png" alt="" />
+          </div>
+          <div
+            class="col-lg-2 pe-4 col-md-4 col-4 px-0 justify-content-center d-flex align-item-center"
+          >
+            <img class="foodie" src="images/partner1.png" alt="" />
+          </div>
+          <div
+            class="col-lg-2 pt-5 pb-5 pt-lg-0 pb-lg-0 pe-4 col-md-4 col-4 px-0 justify-content-center d-flex align-item-center"
+          >
+            <img class="foodie" src="images/partner4.png" alt="" />
+          </div>
+          <div
+            class="col-lg-2 pt-5 pb-5 pt-lg-0 pb-lg-0 pe-4 col-md-4 col-4 px-0 justify-content-center d-flex align-item-center"
+          >
+            <img class="foodie" src="images/partner5.png" alt="" />
+          </div>
+          <div
+            class="col-lg-2 pt-5 pb-5 pt-lg-0 pb-lg-0 pe-4 col-md-4 col-4 px-0 justify-content-center d-flex align-item-center"
+          >
+            <img class="foodie" src="images/partner6.png" alt="" />
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <footer>
+      <div class="footer-logo">
+        <img class="footer-img" src="images/logo.png" alt="resto-logo" />
+        <div class="footer-name">
+          <p>Foodland</p>
+          <p>Restaurant</p>
+        </div>
+      </div>
+      <div class="footer-message">
+        <p>@Foodland Restaurant we serve you</p>
+        <p>Exotic meals</p>
+      </div>
+    </footer>
```

Comment:
Kindly fix the footer and partner section to match the required design 👍 

Your Design
![image](https://user-images.githubusercontent.com/66112760/170843524-f0f52eb6-0935-49f2-adc1-1dd1f6486be9.png)

Figma Design
![image](https://user-images.githubusercontent.com/66112760/170843550-5efdc734-f534-426c-992f-ac7866f15196.png)